querydsl-sql/src/main/java/com/querydsl/sql/AbstractSQLQuery.java

```diff
@@ -67,6 +67,7 @@
     private Object lastCell;
 
     private SQLListenerContext parentContext;
+    private StatementOptions statementOptions = new StatementOptions.Builder().build();
```

Comment:
maybe via StatementOptions.builder()...build()

builder() as a factory method for the builder